### PR TITLE
Add ASP.NET Core, Entity Framework Core, Microsoft Bot Framework, Roslyn and C# 7 articles

### PR DESCRIPTION
**Pull request by forcewake**
1. [Create and deploy an ASP.NET Core Web API to Azure Windows](https://blogs.msdn.microsoft.com/benjaminperkins/2017/01/03/create-and-deploy-an-asp-net-core-web-api-to-azure-windows/)
1. [Paging with Entity Framework Core](http://gunnarpeipman.com/2017/01/ef-core-paging/)
1. [Microsoft Bot Framework - Use Bot Framework for Anytime, Anywhere Access to Application Data](https://msdn.microsoft.com/magazine/mt790202?f=255&MSPPError=-2147217396)
1. [Roslyn - Generate JavaScript with Roslyn and T4 Templates](https://msdn.microsoft.com/magazine/mt790203)
1. [How six lines of code + SQL Server can bring Deep Learning to ANY App](https://blogs.technet.microsoft.com/dataplatforminsider/2017/01/05/how-six-lines-of-code-sql-server-can-bring-deep-learning-to-any-app/)
1. [Deconstruction In C# 7.0](http://www.c-sharpcorner.com/article/deconstruction-in-c-sharp-7-0/)

### CODE REVIEW

File: 2017_January/number-18.md [hunk 1]
```diff
@@ -1,8 +1,13 @@
 # .NET Weekly #18 (09.01 - 15.01)
 
 ## Cloud & Web
+1. [Create and deploy an ASP.NET Core Web API to Azure Windows](https://blogs.msdn.microsoft.com/benjaminperkins/2017/01/03/create-and-deploy-an-asp-net-core-web-api-to-azure-windows/)
```
Comment (RamanBut-Husaim) [inline 2017_January/number-18.md:5]: @forcewake Please, add an empty line between header and an article entry.

Comment (RamanBut-Husaim) [reply 2017_January/number-18.md:5]: @forcewake This is required by the markdown validation in VS Code.

**Review by RamanBut-Husaim — CHANGES_REQUESTED** [review]
@forcewake Please, take a look.

**Comment by forcewake** [comment]
@RamanBut-Husaim 🌵

**Review by RamanBut-Husaim — APPROVED** [review]
@forcewake Thanks a lot!

**Pull request merged**